shell/app/modules/apiManagePlatform/pages/api-market/components/relation.tsx

```diff
@@ -407,7 +407,7 @@ const RelationModal = ({ visible, onCancel, versionInfo, mode }: IProps) => {
                   {state.instanceList.map(({ serviceAddr }) => {
                     return (serviceAddr || []).map((url) => (
                       <Select.Option key={url} value={url}>
-                        <Ellipsis title={url} />
```

Comment:
Ellipsis is not used, delete form the import.
check all the same problem in this pr